### ISSUE TITLE
Would love to implement this on the node-dominos-api main repo 

### ISSUE THREAD

**Issue by RIAEvangelist** [Hollywood/its-pizza-time#32]
When contributors have PRs, if they would supply their address, I would send them a pizza or something upon merge. Does that seem possible?

**Comment by Hollywood** [comment]
Yes! @RIAEvangelist email me, I need to refactor some of this so that it's a bit more dynamic.
